src/javascript/JContent/ContentRoute/ContentLayout/ContentLayout.container.jsx

```diff
@@ -159,17 +158,6 @@ export const ContentLayoutContainer = () => {
     };
 
     useEffect(() => {
-        if (data && data.jcr && data.jcr.nodeByPath) {
-            // When new results have been loaded, use them for rendering.
-            let nodeTypeName = data.jcr.nodeByPath.primaryNodeType.name;
-            let subTypes = ['jnt:page', 'jnt:contentFolder', 'jnt:virtualsite'];
-            let isSub = !subTypes.includes(nodeTypeName);
-            // Sub is not the same as params.sub; refresh and sync up path param state
-            if (isSub !== (params.sub === true)) { // Params.sub needs to be boolean type; else falsy
-                setPath(path, {sub: isSub});
-            }
-        }
```

Comment:
just to confirm, but I didn't see this refactored anywhere. is this not needed anymore (specifically `setPath`)?

Reply:
yes, good catch :-) actually this code was supposed to detect when the `sub` parameter was incorrectly set, but I did not manage to find a real use case. The sub parameter is correctly passed on links, false on pages/folder and true on content. However, it's keeping switching from `sub` to false then back to true when browsing a content (so 2 setPath operations which breaks the navigation history). 
as it's using data.jcr.nodeByPath which is not available anymore, i prefered to get rid of it ..